csunplugged/topics/management/commands/_GlossaryTermsLoader.py

```diff
@@ -23,29 +22,33 @@ def __init__(self, glossary_folder_path, structure_file_path, BASE_PATH):
         super().__init__(BASE_PATH)
         self.structure_file_path = structure_file_path
         self.BASE_PATH = os.path.join(self.BASE_PATH, glossary_folder_path)
+        self.FILE_EXTENSION = ".md"
 
     @transaction.atomic
     def load(self):
         """Load the glossary content into the database."""
+        glossary_slugs = set()
         for filename in listdir(self.BASE_PATH):
-            glossary_file_name = re.search(r"(.*?).md$", filename)
-            if glossary_file_name:
-                glossary_slug = glossary_file_name.groups()[0]
+            if filename.endswith(self.FILE_EXTENSION):
+                glossary_slug = filename[:-len(self.FILE_EXTENSION)]
+                glossary_slugs.add(glossary_slug)
+                glossary_term = GlossaryTerm(
+                    slug=glossary_slug,
+                )
+                glossary_term.save()
+
+        for glossary_slug in glossary_slugs:
+                glossary_term = GlossaryTerm.objects.get(slug=glossary_slug)
                 glossary_file_path = os.path.join(
                     self.BASE_PATH,
-                    filename
+                    glossary_slug + self.FILE_EXTENSION
```

Comment:
much prefer `.format()` rather than concatenation, more consistent across the rest of the loaders